packages/api-kit/tests/utils/signDelegate.ts

```diff
@@ -0,0 +1,22 @@
+import { Signer } from 'ethers'
+import { padHex } from '@safe-global/api-kit/utils/data'
+
+export async function signDelegate(signer: Signer, delegateAddress: string, chainId: string) {
```

Comment:
If this is only a test util perhaps make more sense to add it as a real util in the src/utils folder and use it both in the private method and in the tests. This allow only maintance in 1 place instead 2. WDYT?

Reply:
Agreed, that was the initial approach. I forgot why I changed.